online-inference/stable-diffusion/01-huggingface-secret.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+data:
+  token: Q0hBTkdFTUU=
```

Comment:
Are we supposed to have a token here?

Reply:
Yeah good question what is this?

Reply:
The model has been released publicly, but you are still required to have a HuggingFace account to share your contact info and agree to the license before downloading.

Can be seen [here](https://huggingface.co/CompVis/stable-diffusion-v1-4) in a private/incognito window(assuming you have an HF account and are logged in).

The token is required to help enforce this. 

As for the specific value `Q0hBTkdFTUU=` this just base64 decodes to `CHANGEME` and will not actually work.

Reply:
Aha, I assume that's in the docs how to encode it and apply it.

Reply:
Yes that is specified.